compiler/one-cmds/one-codegen

```diff
@@ -94,15 +95,46 @@ def _get_parser(backends_list):
     return parser
 
 
-def _verify_arg(parser, args):
+def _verify_arg(parser, args, cfg_args):
     """verify given arguments"""
+    if oneutils.is_valid_attr(cfg_args, 'backend') and oneutils.is_valid_attr(
+            cfg_args, 'backends') and oneutils.is_valid_attr(cfg_args, 'command'):
+        parser.error(
+            '\'backend\' option and \'backends\' option cannot be used simultaneously.')
+
+    # Check if given backend from command line exists in the configuration file
+    if oneutils.is_valid_attr(args, 'backend') and oneutils.is_valid_attr(
+            cfg_args, 'backend'):
+        if args.backend != cfg_args.backend:
+            parser.error('Not found the command of given backend')
+
     # check if required arguments is given
     missing = []
-    if not oneutils.is_valid_attr(args, 'backend'):
+    if not oneutils.is_valid_attr(args, 'backend') and not oneutils.is_valid_attr(
+            args, 'backends') and not oneutils.is_valid_attr(
+                cfg_args, 'backend') and not oneutils.is_valid_attr(cfg_args, 'backends'):
         missing.append('-b/--backend')
     if len(missing):
         parser.error('the following arguments are required: ' + ' '.join(missing))
 
+    if oneutils.is_valid_attr(cfg_args, 'backends'):
+        backends = getattr(cfg_args, 'backends').split(',')
+        # check if commands of given backends exist
+        for b in backends:
+            if not oneutils.is_valid_attr(cfg_args, b):
+                parser.error('Not found the command for ' + b)
+
+        # Check if given backend from command line exists in the configuration file
+        if oneutils.is_valid_attr(args, 'backend') and oneutils.is_valid_attr(
+                cfg_args, 'backends'):
+            exists = False
+            for b in backends:
+                if b == args.backend:
+                    exists = True
+                    break
+            if not exists:
```

Comment:
It may be simplified as below.

```suggestion
            if args.backend not in backends:
```

(optional) better to rename backends to cfg_backends, to distinguish it from cmd_backend.